cmd/ethkey/main.go

```diff
@@ -0,0 +1,322 @@
+package main
```

Comment:
New files in cmd/ need the GPLv3 header. Please copy it from cmd/geth/main.go.

Reply:
Done.